### PR TITLE
ports snow golems from tg in preperation for christmas

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
nanotrasen officials have found a living snow man on the ice planet.

snowgolems are a type of golem that are incredibley weak to heat and cant be revived in any form when killed anyway (you can make a farm of these to grind for carrots if your insane) but they also have the two genetic cold related powers (geladkenesis, cryokenesis)
#### Changelog

:cl:  
rscadd: Ports Snow Golem
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Who doesn't love space proof species

**Comment by boodaliboo** [comment]
> Who doesn't love space proof species

golems are fun

**Comment by boodaliboo** [comment]
> Who doesn't love space proof species

also this bridges a gap between genetics and xenobio

File: code/modules/holiday/halloween.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /obj/structure/closet/proc/set_spooky_trap()
 	if(prob(0.1))
 		trapped = INSANE_CLOWN
-		return
+		returncode/modules/holiday/halloween.dm
```
Comment (ThatLing) [inline code/modules/holiday/halloween.dm]: wtf

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/holiday/halloween.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/holiday/halloween.dm]: fixed

**Pull request merged**